editoast/editoast_derive/src/error.rs

```diff
@@ -37,15 +37,21 @@ pub fn expand_editoast_error(input: &DeriveInput) -> Result<TokenStream> {
         _ => return Err(Error::custom("EditoastError: Only enums are supported.")),
     };
     let variants = parse_variants(enum_data)?;
+    let default_status = options.default_status.unwrap_or(DEFAULT_STATUS_CODE);
+    let get_statuses = expand_get_statuses(&variants, default_status)?;
 
-    let get_statuses = expand_get_statuses(
-        &variants,
-        options.default_status.unwrap_or(DEFAULT_STATUS_CODE),
-    )?;
     let contexts = expand_contexts(&variants);
     let get_types = expand_get_types(&variants, base_id.clone());
 
+    let error_definitions = TokenStream::from_iter(
+        variants
+            .iter()
+            .map(|v| parse_error_definition(base_id.clone(), default_status, v)),
```

Comment:
We can use a reference here.

```suggestion
            .map(|v| parse_error_definition(&base_id, default_status, v)),
```

Reply:
done